client/src/components/TrailShowPage.js

```diff
@@ -23,6 +25,15 @@ const TrailShowPage = (props) => {
     getTrail()
   }, [])
 
+  const getReviews = trail.reviews.map((review) => {
+    return (
+      <ul>
+        <p>Comment: {review.comment}</p>
+        <p>Rating: {review.rating}</p>
+      </ul>
```

Comment:
Children of an `ul` unordered list should be `li` list items, we can update this to be a `div` that wraps around the multiple elements

Reply:
Done.